Cirrious.FluentLayout/FluentLayout.cs

```diff
@@ -153,7 +154,7 @@ private void ThrowIfSecondItemAlreadySet()
 
         public IEnumerable<NSLayoutConstraint> ToLayoutConstraints()
         {
-            var constraint = NSLayoutConstraint.Create(
+            Constraint = NSLayoutConstraint.Create(
```

Comment:
While this method only seems to be returning a collection of one constraint, this change doesn't really sit well with me. The `Constraint` property would only be set as a side effect of calling `ToLayoutConstraints()`, which itself technically returns a collection of constraints.

Couldn't you achieve what you want by using the `NSLayoutConstraint` instance(s) returned by this method instead of having one of them exposed as a member of this class?